launcher/Application.cpp

```diff
@@ -410,6 +413,47 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
                 " " "|" " "
                 "%{if-category}[%{category}]: %{endif}"
                 "%{message}");
+      
+        bool foundLoggingRules = false;
+        
+        auto logRulesFile = QStringLiteral("qtlogging.ini");
+        auto logRulesPath = FS::PathCombine(dataPath, logRulesFile);
+        
+        qDebug() << "Testing" << logRulesPath << "...";              
+        foundLoggingRules = QFile::exists(logRulesPath);
+
+        // search the dataPath()
+        
+        if(!foundLoggingRules && !isPortable() && dirParam.isEmpty()) {
+            logRulesPath = QStandardPaths::locate(QStandardPaths::AppDataLocation, logRulesFile);
+            if(!logRulesPath.isEmpty()) {
+                qDebug() << "Found" << logRulesPath << "...";
+                foundLoggingRules = true;
+            }
+        }
+
+        if(!QFile::exists(logRulesPath)) {
```

Comment:
From what I can tell, foundLoggingRules can only be true if a file has already been found. So I guess we can just use that?

```suggestion
        if(!foundLoggingRules) {
```